lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -4615,12 +4634,14 @@ def model_apply_prm_baseline_window_to_wall_ratio(model, climate_zone, wwr_build
         wwr_lim = 40.0
       end
 
+      wwr_lim = model_adjust_wwr_based_on_bat(wwr_lim, bat, [vals['wwr_nr'], vals['wwr_res'], vals['wwr_sh']])
+
       # Check against WWR limit
       vals['red_nr'] = vals['wwr_nr'] > wwr_lim
       vals['red_res'] = vals['wwr_res'] > wwr_lim
       vals['red_sh'] = vals['wwr_sh'] > wwr_lim
 
-      # Stop here unless windows need reducing or increasing if
+      # Stop here unless windows need reducing or increasing or if
       # following the stable baseline approach
       return true, base_wwr unless (vals['red_nr'] || vals['red_res'] || vals['red_sh']) || template == '90.1-PRM-2019'
```

Comment:
We've been trying to get away from using template as the trigger for stable baseline features. I think a good alternative might be to add a method named "stable_baseline?", and use inheritance to give it a value of "true" for 90_1_prm and "false" in Standards.Model.rb. Also applies to line 4619 above.